### PR TITLE
Add readiness probes for rollout traffic routing

### PR DESCRIPTION
**Pull request by camallen**
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes

only route traffic to the new pods once they are ready to accept it. I tested boot on my dev machine and it was ~25s so going with 20 for production.

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: kubernetes/deployment-production.tmpl [hunk 1]
```diff
@@ -386,6 +402,7 @@ spec:
       containers:
         - name: panoptes-production-redis
           image: redis:6.0.5
+          args: ['--save ""', '--appendonly no']
```
Comment (adammcmaster) [inline kubernetes/deployment-production.tmpl:405]: Did you mean to include this here? Don't we want redis data to be persistent for Panoptes?

File: kubernetes/deployment-staging.tmpl [hunk 2]
```diff
@@ -330,6 +346,7 @@ spec:
       containers:
         - name: panoptes-staging-redis
           image: redis:6.0.5
+          args: ['--save ""', '--appendonly no']
```
Comment (adammcmaster) [inline kubernetes/deployment-staging.tmpl:349]: Did you mean to include this here? Don't we want redis data to be persistent for Panoptes?

File: kubernetes/deployment-production.tmpl (hunk 1 above)
Comment (camallen) [reply kubernetes/deployment-production.tmpl:405]: wtf!? absolutely not. bad me
